### ISSUE TITLE
Bot registration

### ISSUE THREAD

**Issue by Forket** [Forket/connect2sa.co.za_01#12]
I remember you asked about how to prevent bots in registering at your site.
If this question is open - I can help you.

**Comment by heinvv** [comment]
Yes, this question is still open. But I'd like to widen the scope of this question a bit. You can expect an update about this tomorrow.

**Comment by Forket** [comment]
OK. will wait

**Comment by heinvv** [comment]
Please find our ideas in the word document attached
[20151130 User registration.docx](https://github.com/Forket/connect2sa.co.za_01/files/47056/20151130.User.registration.docx)

**Comment by heinvv** [comment]
It is very important for us that everything on the website is Responsive. The login popup isn't responsive right now. Can you please fix that for us?

**Comment by Forket** [comment]
Yes. It's possible. Will write you back with this issue later. Now focused
on completed other issues which you plan to have in 2 days.

Thanks and regards,

*Aleksandr Sen*

CEO | Expedition PM

Mobile: +48 532 292 764

Skype: aleksandr.sen

Email: alex@sen.pm

Social: LinkedIn <http://ua.linkedin.com/in/aleksandrsen/> | Google+
<https://www.google.com/+AleksandrSen>



*EXPEDITION.PM <http://www.expedition.pm/> | KNOWLEDGEMAP.PM
<http://www.knowledgemap.pm/>*

2015-11-30 16:29 GMT+01:00 heinvv <notifications@github.com>:

**Comment by heinvv** [comment]
Perfect

**Comment by Forket** [comment]
Forget about captcha. It only interrupts user from normal site using. I have one solution that will not disturb users :) and you will not have spam registrations and comments.
If you agree - let me know and I will do everything.

**Comment by heinvv** [comment]
Sounds great.

Can you please send me a bit more information about this solution, before I
make a solution?

Can you send me a link about it?

**Comment by Forket** [comment]
I'll connect your site to analytics server, which uses very complicated algorithm for detecting spam.
I've just connected your site for testing. So I'll allow registration and we will see.
Am I right that you had a lot of span at http://www.connect2sa.co.za/rigel/ site?

**Comment by heinvv** [comment]
Okay, cool.

There was a bit of spam indeed

-----Original Message-----

**Comment by Forket** [comment]
>There was a bit of spam indeed
It's before i've connected your site to the system. 
Please let me know if you will see spam. Also for now I know you have some IP filter. Could you please switch it off? and we will see result.

**Comment by heinvv** [comment]
We don't have ip filter

-----Original Message-----

**Comment by Forket** [comment]
OK. can u tell me if you will see some spam in site? If there will not be any spam i'll configure plugin to use this solution further.

**Comment by heinvv** [comment]
I'll  let you know 

-----Original Message-----

**Comment by heinvv** [comment]
I don't see any spam coming in. 
Can you please continue working on the rest of the document User Registration?

**Comment by Forket** [comment]
I'm already working on it. I'll see feedback on related issues.

**Comment by Forket** [comment]
This is result from this week. Blocked spam

![snapshot n001_2015-12-12_11-32-56](https://cloud.githubusercontent.com/assets/6735288/11761463/12c29972-a0c4-11e5-82fd-df5983b9525d.png)

**Comment by heinvv** [comment]
:)

-----Original Message-----
